openedx/core/djangoapps/user_api/accounts/settings_views.py

```diff
@@ -25,7 +25,8 @@
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 from openedx.core.djangoapps.user_api.accounts.toggles import (
     should_redirect_to_account_microfrontend,
-    should_redirect_to_order_history_microfrontend
+    should_redirect_to_order_history_microfrontend,
+    should_show_linked_accounts_tab
```

Comment:
trailing comma

Reply:
@andrey-canon why trailing comma? at the end of the tuple?

Reply:
https://coderwall.com/p/4fvcaa/trailing-comma-in-python-data-structures

Reply:
ok, I understand it, but in the past, it didn't have the trailing comma. But you are right in that way using the tuple in not one line. https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Reply:
in the past, it didn't have the trailing comma and that's why the git diff shows two changes instead of one